.travis.yml

```diff
@@ -2,30 +2,40 @@ language: go
 node_js: '8.3'
 sudo: required
 dist: trusty
+
 go:
 - 1.10.x
+
 before_install:
 - openssl aes-256-cbc -K $encrypted_56d12e7424aa_key -iv $encrypted_56d12e7424aa_iv
   -in .env.enc -out .env -d
 - nvm install 8.3
 - npm install -g npm@6.x
+- sudo apt-get update
```

Comment:
You add doing `apt-get update and installing python in every single build.

That seems a little bit too much.

How much time is this adding to the build?

Reply:
Just 4 secs.
<img width="1317" alt="screen shot 2018-11-05 at 11 02 12 am" src="https://user-images.githubusercontent.com/13935994/48009738-476b8d00-e0ea-11e8-84e8-a3e0fa144e7d.png">